pkg/partitioner/disk.go

```diff
@@ -17,13 +17,14 @@ limitations under the License.
 package partitioner
 
 import (
-	"errors"
 	"fmt"
 	"os"
 	"regexp"
 	"strings"
 	"time"
 
+	"github.com/pkg/errors"
```

Comment:
this has been archived

Reply:
oh! I didn't know that. Will use `fmt.Errorf` or something. Preparing a fix